mapflow/mapflow.py

```diff
@@ -1792,6 +1800,23 @@ def submit_processing_rating_callback(self, response: QNetworkReply) -> None:
         )
         self.update_processing_current_rating()
 
+    def enable_rating_submit_button(self, radio_checked: Optional[bool]) -> None:
+        status_ok = False
+        radio_button_checked = False
+        row = self.dlg.processingsTable.currentRow()
+        if not row >= 0:
+            return
+        if self.dlg.processingsTable.item(row, self.config.PROCESSING_TABLE_STATUS_COLUMN_INDEX).text() == 'OK':
```

Comment:
It is better to use model rather than view as a source of data.
Model in this case is self.processings, which holds class instances of Processing.

Of course, we need to retrieve processing by ID taken from table, but i think that the single column that we navigate by is better than to access any column.

Reply:
fixed